dotnet-interactive/resources/dotnet-interactive.js

```diff
@@ -1,62 +1,58 @@
 ﻿define({
```

Comment:
I recommend you diff this file with whitespace disabled.  The shape was changed so that everything is essentially indented one level deeper.